src/app/child-dev-project/notes/model/note.ts

```diff
@@ -43,29 +39,15 @@ export class Note extends Entity {
     return this.warningLevel;
   }
 
-  // TODO: color logic should not be part of entity/model but rather in the component responsible for displaying it
-  public getColor() {
-    if (this.warningLevel === WarningLevel.URGENT) {
-      return WarningLevelColor(WarningLevel.URGENT);
-    }
-    if (this.warningLevel === WarningLevel.WARNING) {
-      return WarningLevelColor(WarningLevel.WARNING);
-    }
+  // TODO: What is the purpose of this function?
+  // public getColorForId(entityId: string) {
+  //   if (this.isMeeting() && !this.isPresent(entityId)) {
```

Comment:
for meeting-type notes, the color in the notes section of a child depends on whether the child is present or absent for this note. Otherwise, the highlight color only depends on the note type